app/tests/test_record.py

```diff
@@ -232,26 +295,89 @@ def test_record_download_record_with_citeable_reference(
 
         html = response.data.decode()
 
-        expected_download_html = f"""
-        <div class="rights-container">
-            <h3 class="govuk-heading-m govuk-heading-m__rights-header">Rights to access</h3>
-            <a href="/download/{file.FileId}"
-                class="govuk-button govuk-button__download--record"
-                data-module="govuk-button">Download record</a>
-            <p class="govuk-body govuk-body--download-filename">
-                The downloaded record will be named<br>
-                <strong>{download_filename}</strong>
-            </p>
-            <p class="govuk-body govuk-body--terms-of-use">
-                Refer to <a href="/terms-of-use" class="govuk-link govuk-link--ayr">Terms of use.</a>
-            </p>
-        </div>
+        soup = BeautifulSoup(html, "html.parser")
+        button = soup.find("a", string="Download record")
+
+        assert button is None
+
+    def test_record_aau_user_with_perms_can_download_record_without_citeable_reference(
+        self, client: FlaskClient, mock_all_access_user, record_files
+    ):
+        """
+        Given a File in the database
+        When an all permitted access user with request to view the record page
+        Then the response status code should be 200
+        And the HTML content should see record download component
+        on the page
         """
+        file = record_files[4]["file_object"]
+        mock_all_access_user(client, can_download=True)
+
+        response = client.get(f"{self.route_url}/{file.FileId}")
+
+        assert response.status_code == 200
+
+        html = response.data.decode()
+
+        expected_download_html = (
+            expected_download_html_without_citeable_reference(file.FileId)
+        )
 
         assert_contains_html(
             expected_download_html, html, "div", {"class": "rights-container"}
         )
 
+    def test_record_aau_user_with_perms_can_download_record_with_citeable_reference(
```

Comment:
not sure if we need this extra test at this level, seems quite edge casey, as its just about the citable ref and we have tests for that already